### PR TITLE
Talks page

### PR DESCRIPTION
**Pull request by MozixReality**
1. fetch talk & tutorial pages
2. readme change pip to pipenv

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 Pycon_year = "2016"
 Pycon_url = "https://tw.pycon.org"
 Written_url = []
+Visited_page = []
```
Comment (josix) [inline main.py:13]: It is a better habit for naming variable following [PEP 8 coding style convention](https://www.python.org/dev/peps/pep-0008/). For example, the constant variable should be written in all capital letters with underscores separating words, i.e. PYCON_YEAR, PYCON_URL, WRITTEN_URL...

File: README.md [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
 
 ```
-    python main.py -y year
+    python3 main.py -y year
 ```
 
 where year can just in 2016 - 2020
 
-## Utility Models
+## Pipenv
```
Comment (josix) [inline README.md:13]: IMO, we should have a more meaningful title like `Setup development environment`, `Installing dependencies`

File: main.py (hunk 1 above)
Comment (josix) [inline main.py:13]: It is a better habit for naming variable following PEP 8 coding style convention. For example, [the constant](https://www.python.org/dev/peps/pep-0008/#constants) variable should be written in all capital letters with underscores separating words, i.e. `PYCON_YEAR`, `PYCON_URL`, `WRITTEN_URL`... and [the variable](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) should be written in lowercase, with words separated by underscores

File: main.py [hunk 3]
```diff
@@ -84,11 +84,20 @@ def img(soup):
 
 def GetPage(path):
```
Comment (josix) [inline main.py:85]: Same as above, we should follow PEP8 coding style convention to name function name in lowercase, with words separated by underscores. [Here is the ref](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

File: main.py [hunk 4]
```diff
@@ -84,11 +84,20 @@ def img(soup):
...
     # filter our target path
-    if path[0] != '/':
+    if path[0] != '/' or path in Visited_page:
         return
+    Visited_page.append(path)
     print(path)
```
Comment (josix) [inline main.py:90]: Not sure if this is necessary for keep this printing remain. We can remove it if it don't bring any helpful information or make this printing info more completely as logging message by using logging tool like [loguru](https://github.com/Delgan/loguru), [built-in logging module](https://docs.python.org/3/library/logging.html), etc.

File: main.py [hunk 5]
```diff
@@ -84,11 +84,20 @@ def img(soup):
...
     request = requests.get(Pycon_url + path)
```
Comment (josix) [inline main.py:91]: We cloud introduce [pathlib](https://docs.python.org/3/library/pathlib.html) here for helping us dealing with the path related operation

**Review by josix — CHANGES_REQUESTED** [review]

**Comment by MozixReality** [comment]
review file change

**Review by josix — APPROVED** [review]

**Comment by josix** [comment]
Hello @MozixReality, thanks for your work and sorry for the late review. I'll merge this PR and we will have some refactor works to do.

**Pull request merged**